app/services/downstream_service.rb

```diff
@@ -18,6 +18,11 @@ def self.update_draft_content_store(downstream_payload)
       message = "Can only send draft, published and unpublished items to draft content store"
       raise DownstreamInvalidStateError.new(message)
     end
+    if downstream_payload.state != "draft" && draft_at_base_path?(downstream_payload.base_path)
+      message = "Can't send #{downstream_payload.state} item to draft content store, as there is a draft occupying the same base path"
+      raise DownstreamInvalidStateError.new(message)
```

Comment:
So Kevin and I discussed this and a few other things for a while.

I think this is ok, but it would be best just to log these errors, and not send them to Errbit.